DqtApi/src/DqtApi/DataStore/Crm/DataverseAdapter.SetNpqQualification.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using DqtApi.DataStore.Crm.Models;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Messages;
+
+namespace DqtApi.DataStore.Crm
+{
+    public partial class DataverseAdapter
+    {
+        public async Task<SetNpqQualificationResult> SetNpqQualification(SetNpqQualificationCommand command)
+        {
+            var (r, _) = await SetNpqQualificationImpl(command);
+            return r;
+        }
+
+        internal async Task<(SetNpqQualificationResult Result, ExecuteTransactionRequest TransactionRequest)> SetNpqQualificationImpl(SetNpqQualificationCommand command)
+        {
+            var txnRequest = new ExecuteTransactionRequest()
```

Comment:
Does we need a `ExecuteTransactionRequest` here? Looks like we're only ever having a single child request.